src/commons/BrowserUtils.ts

```diff
@@ -229,6 +238,16 @@ export namespace BrowserUtils {
     );
   }
 
+  /**
+   * @param selector - element selector
+   */
+  export function isEnabled(selector: string): boolean {
+    Reporter.debug(`Is element enabled '${selector}'`);
+    isExist(selector);
+
+    return $(selector).isEnabled();
```

Comment:
why not  expectEnabled/expectDisabled implemented with waitForEnabled ?
there is big chance that it will return false state

Reply:
expectEnabled/expectDisabled has a different purpose then what i need 
I need a fast, no wait, true false answer to is button is enabled. if it is i click it again if not i stop clicking it.